server/cpho/views/indicator_data.py

```diff
@@ -232,6 +232,7 @@ def age_group_formset(self):
         for form in fs:
             # new unsaved instances' save() crash when no dimension type is specified
             form.instance.dimension_type = age_dimension
+            form.instance.period = self.period
 
```

Comment:
There was previously a bug where you would add new age group data, but not see it show up. It was in fact creating records, but since they weren't linked to any period they would not appear. We may want to add a non-null constraint on the period field of indicator-datum